Serializer/Jms/Handler/ResourceHandler.php

```diff
@@ -85,6 +86,7 @@ private function doSerializeResource(Resource $resource, $depth = 0)
         }
 
         $data['path'] = $resource->getPath();
+        $data['node_name'] = PathHelper::getNodeName($data['path']);
```

Comment:
I assume this is the text that appears beside the node in the tree, if so would it make more sense to call this a `label` ? 

Would the tree use the resource name by default?